config/maps.txt

```diff
@@ -40,7 +40,7 @@ map donutstation
 endmap
 
 map runtimestation
-	disabled
+	minplayers 0
```

Comment:
don't think you wanted to commit that

Reply:
You can load runtime station only by commenting #define LOWMEMMODE which doesn't load lavaland so its quicker

Reply:
Ah thanks, though im not sure how to not commit that as any change i make gets added into the commit and reverting it also reverts the file